configuration/boot_folder.md

```diff
@@ -35,6 +35,26 @@ Contains many configuration parameters for setting up the Pi. See [the `config.t
 
 Some text-based housekeeping information containing the date and git commit ID of the distribution.
 
+### ssh or ssh.txt
+
+When this file is present, SSH will be enabled on boot. The contents don't matter, it can be empty. SSH is otherwise disabled by default.
+
+### wpa_supplicant.conf
+
+This is the file to configure wireless network settings (if the hardware is capable of it). Edit the country code and the network part to fit your case.
```

Comment:
This should probably link to https://github.com/raspberrypi/documentation/blob/master/configuration/wireless/headless.md , rather than repeating the  `wpa_supplicant.conf` example file?

Reply:
Yes, I think that would be the best approach. Otherwise I'm happy with the content of this PR.

Reply:
Thank you, I hadn't seen that page.